core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -464,9 +464,10 @@ private[spark] object UIUtils extends Logging {
     val completeWidth = "width: %s%%".format(ratio)
     // started + completed can be > total when there are speculative tasks
     val boundedStarted = math.min(started, total - completed)
-    val startWidth = "width: %s%%".format((boundedStarted.toDouble/total)*100)
+    val startRatio = if (total == 0) 0.0 else (boundedStarted.toDouble/total)*100
```

Comment:
nit: Now that we touch this code, can we fix the style too like as follows?
```
val startRatio = if (total == 0) 0.0 else (boundedStarted.toDouble / total) * 100

Reply:
Done